resume.html

```diff
@@ -0,0 +1,515 @@
+            <div class="row normal-row">
+                <div class="col-2 gray-div">
+            
+                </div>
+                <div class="col-md-2 col-4 gray-div left-article">
+                    Experience
+                </div>
+                <div class="col-md-4 col gray-div">
+                
+                </div>
+                <div class="col-md-2 col-4 gray-div right-article">
+                    <input class="res-btn" type="button" value="DOWNLOAD CV">
+                </div>
+                <div class="col gray-div">
+                
+                </div>
+            </div>
+            <div class="row normal-row">
+                <div class="col-2 gray-div">
+                    
+                </div>
+                <div class="col-4">
+                
+                </div>
+                <div class="col-4">
+                
+                </div>
+                <div class="col-2 gray-div">
+                
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-2 gray-div">
+            
+                </div>
+                <div class="col-8 sub-title">
+                    Jan 2015 - Jan 2019
+                </div>
+                <div class="col-2 gray-div">
+            
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-2 gray-div">
+                    
+                </div>
+                <div class="col-4">
+                    <h5>Oracle Techno-Functional Consultant</h5>
+                    <p>AL_Rowad for IT Solutions</p>
+                    <p>UAE - Abu Dhabi</p>
+                </div>
+                <div class="col-4">
+                    <p> Fulfilled full core HR and Payroll implementation projects for two large-scale companies.</p>
+                    <p> Supported and enhanced the processes of core HR and Payroll for many organizations.</p>
+                    <p> Designed customized solutions within the Oracle ERP system to help resolve specific business issues.</p>
+                    <p> Built a lot of solutions to help meet clients’ HR and Payroll requirements such as absence management, benefits and
+                    compensations. Completed tasks to the full satisfaction of clients.</p>
+
+                </div>
+                <div class="col-2 gray-div">
+            
+                </div>
+            </div>
+            <div class="row narrow-row">
+                <div class="col gray-div">
+            
+                </div>
+            </div>
+            <div class="row normal-row">
+                <div class="col-2 gray-div">
+            
+                </div>
+                <div class="col-4">
+            
+                </div>
+                <div class="col-4">
+            
+                </div>
+                <div class="col-2 gray-div">
+            
+                </div>
+            </div>
```

Comment:
This is not the correct way to create empty space with Bootstrap: see my other comment about about using Bootstrap's `offset` classes and also [the docs](https://getbootstrap.com/docs/4.0/layout/grid/#offsetting-columns)